packages/website/docs/state.md

```diff
@@ -52,24 +52,33 @@ You can also manually update the state using setters. It's useful to implement c
 For example, let's say you want to let users fill the search input with the value of a suggestion by clicking or tapping it. You can use the [`setQuery`](state#setquery) setter provided by [`getSources`](sources#getsources) to attach an event when clicking the tap-ahead button and manually set the query.
 
 ```js
+/** @jsx h */
+import { h } from 'preact';
+import { autocomplete } from '@algolia/autocomplete-js';
+
 autocomplete({
   // ...
-  getSources({ query, setQuery, refresh }) {
+  getSources({ setQuery, refresh }) {
     return [
       {
         // ...
         templates: {
-          item({ item, root }) {
-            const tapAheadButton = document.createElement('button');
-
-            tapAheadButton.addEventListener('click', (event) => {
-              event.stopPropagation();
-
-              setQuery(item.query);
-              refresh();
-            });
-
-            root.appendChild(tapAheadButton);
+          item({ item }) {
+            return (
+              <div>
```

Comment:
In [`a725011` (#401)](https://github.com/algolia/autocomplete.js/pull/401/commits/a725011b8d3ab9ad230501cf9023d157c3169af4#r561716568) I added class names because they will be conventions for Autocomplete themes and plugins. Not a big deal here I guess.

Reply:
I kept them out for terseness and because we don't document the themes here. Now I assume this is easier for someone to remove classes they don't need than to figure out where to add classes they need if they use the theme.

I noticed that one class name has "aa" twice ("aa-aa-ItemSourceIcon") and one had no "aa" prefix ("ItemContent"). Is it on purpose?

Reply:
No that's a mistake indeed.

Reply:
So that would be:

- `aa-ItemContent`
- `aa-ItemSourceIcon`
- `aa-ItemTitle`
- `aa-ItemActionButton`

Right?

Reply:
Yes

Reply:
Done in 5de5926.